src/bindings/rust/README.md

```diff
@@ -8,6 +8,21 @@
 
 Rust bindings for libelektra.
 
+## Build
+
+To build the bindings explicitly as part of the elektra build process, we add the option `rust` to `-DBINDINGS`. After [building libelektra](../../../doc/COMPILE.md), we can now add the `elektra` crate to the dependencies. The exact paths depends on your system.
```

Comment:
The current status is `- infos/status = experimental maintained`. I think it will be included by default if I remove experimental. Should I do that?

Reply:
I would be okay with that. You can also do it later when you are more confident.